src/runtime/providers/imagekit.ts

```diff
@@ -9,21 +9,136 @@ const operationsGenerator = createOperationsGenerator({
     height: 'h',
     format: 'f',
     quality: 'q',
-    background: 'bg'
+    bg: 'bg',
+    background: 'bg',
+    aspectRatio: 'ar',
+    x: 'x',
+    y: 'y',
+    xc: 'xc',
+    yc: 'yc',
+    oix: 'oix',
+    oiy: 'oiy',
+    oixc: 'oixc',
+    oiyc: 'oiyc',
+    focus: 'fo',
+    radius: 'r',
+    border: 'b',
+    rotate: 'rt',
+    blur: 'bl',
+    named: 'n',
+    overlayX: 'ox',
+    overlayY: 'oy',
+    overlayFocus: 'ofo',
+    overlayHeight: 'oh',
+    overlayWidth: 'ow',
+    overlayImage: 'oi',
+    overlayImageTrim: 'oit',
+    overlayImageAspectRatio: 'oiar',
+    overlayImageBackground: 'oibg',
+    overlayImageBorder: 'oib',
+    overlayImageDPR: 'oidpr',
+    overlayImageQuality: 'oiq',
+    overlayImageCropping: 'oic',
+    overlayImageCropMode: 'oicm',
+    overlayText: 'ot',
+    overlayTextFontSize: 'ots',
+    overlayTextFontFamily: 'otf',
+    overlayTextColor: 'otc',
+    overlayTextTransparency: 'oa',
+    overlayTextTypography: 'ott',
+    overlayBackground: 'obg',
+    overlayTextEncoded: 'ote',
+    overlayTextWidth: 'otw',
+    overlayTextBackground: 'otbg',
+    overlayTextPadding: 'otp',
+    overlayTextInnerAlignment: 'otia',
+    overlayRadius: 'or',
+    progressive: 'pr',
+    lossless: 'lo',
+    trim: 't',
+    metadata: 'md',
+    colorProfile: 'cp',
+    defaultImage: 'di',
+    dpr: 'dpr',
+    effectSharpen: 'e-sharpen',
+    effectUSM: 'e-usm',
+    effectContrast: 'e-contrast',
+    effectGray: 'e-grayscale',
+    original: 'orig'
   },
   valueMap: {
     fit: {
       cover: 'maintain_ratio',
       contain: 'pad_resize',
       fill: 'force',
       inside: 'at_max',
-      outside: 'at_least'
+      outside: 'at_least',
+      extract: 'extract',
+      pad_extract: 'pad_extract'
     },
-    background (value = '') {
```

Comment:
This was added by someone about 2 weeks ago from today(14/6/21). It is copied from Imgix docs as it is, see this [PR](https://github.com/nuxt/image/pull/304).
Since we do not use `#` in our SDKs or IK, I don't think we should allow it for nuxt/image as well. It should be consistent with our SDKs. To be even more clear we have specified in readme not to use `#` while using hex for color params.

Also, if we have it for `background`. We will have to add it for other hex input color params as well. Which is just cascadingly adding something not needed all over our other parameters. 

The correct way to use IK params should not be decided by someone who was adding fillers for our docs, but by how our other SDKs etc. work.

Additionally, there is an ambiguity/problem with their background prop for which an issue needs to be raised.
`background` works as a prop itself and it is missing in their docs. And, `background` does not work inside `modifiers` prop, so we use `bg` in its place.

Reply:
Let's explain this reasoning in our final PR to them.

Reply:
Let's keep what is already there. It's good. I have myself provided `#` a couple of times by mistake. If SDK can remove it, perfect!

Reply:
Ok, then we can add a function to remove `#` and to convert hex letters to uppercase for all color parameters.